spec/spec_helper.rb

```diff
@@ -80,7 +80,8 @@
         to_return(:status => 200,
                   :body => File.open(SPEC_ROOT + '/fixtures/request_options_no_hold.json'))
 
-
+    stub_request(:post, /.*\.exlibrisgroup\.com\/almaws\/v1\/bibs\/foo\/holdings\/hold\/items\/pid\/requests/).
```

Comment:
I'd only suggest generalizing this to `/.*\.exlibrisgroup\.com\/almaws\/v1\/bibs\/.*\/holdings\/.*\/items\/.*\/requests/`, in case we decide to add more test but use different dummy data.

Reply:
@bibliotechy That makes sense.  I updated it to be more generalized.